injector/CollectVarDependencies/CollectVarDependencies.py

```
@@ -0,0 +1,20 @@
+import ast
+import copy
```

Ruff (0.11.9)

`copy` imported but unused

Remove unused import: `copy`

(F401)